lib/Notification/index.js

```diff
@@ -28,23 +32,34 @@ const Notification = ({
       className={`notification notification--${level} ${alertClasses} ${className}`}
       {...rest}
     >
-      {children}
+      <div className="notification__content">{children}</div>
+      {onClose && (
+        <Button
+          onClick={onClose}
+          types={['icon-only']}
+          className="notification__close"
+        >
+          <Icon name="cross" />
+        </Button>
```

Comment:
🍏 Should either of these things have a `title`? It might make testing with it in the futur easier.

Reply:
Excellent point! will add